src/components/core/DpUpload/DpUploadedFile.vue

```diff
@@ -8,7 +8,7 @@
     </span>
     <span v-if="isImage">
       <img
-        :src="Routing.generate('core_file', { hash: file.hash })"
+        :src="getFileByHash(file.hash)"
```

Comment:
I think we should try to avoid using any type of generate route within demosplan-ui. We better should pass the generated src.

Reply:
Here no more using `Routing.generate` within demosplan-ui!

Reply:
`getFileByHash` does and is part of demosplan-ui ;-)

Reply:
but we can leave it like it is by now.

Reply:
@salisdemos this request is needed to show image thumbs of images that are already uploaded. how would you solve this without api requests?

Reply:
As I said. I am fine with leaving like it iss - by now.
But we have to find a way to handle that. I think we don't need dpApi here in the end, if we build a way to pass the generated thumbnails. 
But if we leave it like it is now, who ever uses this has to use the logic from "getFileByHash" to use the uploader / generate a image-url. That feels too coupled for me.